src/main/java/com/example/carsharingservice/service/impl/RentalServiceImpl.java

```diff
@@ -26,7 +26,7 @@ public Rental createNewRental(Car car, User user, LocalDate returnDate) {
         rental.setUser(user);
         rental.setRentalStart(LocalDate.now());
         rental.setRentalReturn(returnDate);
-        carService.takeCarFromInventory(car.getId());
+        // carService.takeCarFromInventory(car.getId());
```

Comment:
why comment is there

Reply:
Because I don't understand where he takes this method. Is it the same "removeCarFromInventory"?

Reply:
i suppose yes